backend/vault.go

```diff
@@ -154,14 +154,15 @@ func (c *client) renewalLoop() {
 		c.logger.Info("trying to login to vault again")
 		if err = c.vaultLogin(); err != nil {
 			vMetrics.updateVaultLoginErrorsTotalMetric()
```

Comment:
should we log an error here? that we couldn't manage to login?

Reply:
I agree here :)